partcad/src/partcad/project_factory_git.py

```
@@ -150,7 +157,9 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
                 # Clone the repository if it's not cached yet.
                 try:
                     pc_logging.info("Cloning the GIT repo: %s" % self.import_config_url)
-                    repo = Repo.clone_from(repo_url, cache_path)
+                    repo = Repo.clone_from(repo_url, cache_path, env={
+                        "GIT_CONFIG_PARAMETERS": git_config_params
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)